client.py

```diff
@@ -1,53 +1,67 @@
-#
-#   Request-reply client in Python
-#   Connects REQ socket to tcp://localhost:5559
-#   Sends "Hello" to server, expects "World" back
-#
 import tqdm as tqdm
 import zmq
 import numpy as np
 from utils import send_array
 from numpy import random
 
 
+class Client:
+    def __init__(self, addr="tcp://localhost:5559", port=8192):
```

Comment:
The address should just be `*`, and port should be passed as an `int` to the constructor.